app/src/androidTest/java/com/android/unio/end2end/EndToEndTest.kt

```diff
@@ -62,19 +62,17 @@ open class EndToEndTest : FirebaseEmulatorFunctions {
               }
 
               override fun onResponse(call: okhttp3.Call, response: okhttp3.Response) {
-                if (response.body == null) {
+                if (response.body == null || !response.body!!.string().contains("Ok")) {
```

Comment:
Haha I have no idea how to change that but isn't there is a Json response ? That means we could check that the response is OK without consuming the response body. If not that seems really fine

Reply:
The body isn't a JSON response, it is is simply the string `Ok` when the emulator is successfully running. Thus, it should be okay to consume the body as it will not be used anywhere else.